docs/configuration.md

```diff
@@ -32,434 +32,124 @@ spec:
 
 | Parameter | Description |
 | --------- | ----------- |
-| agent.additionalAnnotations | AdditionalAnnotations provide annotations that will be added to the Agent Pods. |
```

Comment:
i wonder if we should keep a copy of this table for v1alpha1 somewhere

Reply:
Generating both to different files now

Reply:
thanks!